app/com/arpnetworking/metrics/portal/reports/ReportExecutionContext.java

```diff
@@ -174,6 +176,16 @@
                                         .setTimeRange(timeRange),
                                 report.getTimeout()
                         )
+                        .whenComplete((res, err) -> {
+                            if (err != null) {
+                                LOGGER.warn()
```

Comment:
Does the error from this future get swallowed here? It seems like we should leave the logging to whomever is waiting on the future's result (e.g. don't log and throw).